task_master.go

```diff
@@ -82,6 +82,10 @@ type TaskMaster struct {
 		StateChangesOnly() bool
 		Alert(channel, message string, level AlertLevel) error
 	}
+	SnmpService interface {
+		StateChangesOnly() bool
+		Alert(trap [][]interface{}, level AlertLevel) error
```

Comment:
Do snmp traps support a `message` concept? Meaning can they contain a short string of some short or do they only support the alert level? Also how do you know the context of the alert? For example, say you are triggering alerts on CPU usage per host, when the NMS receives the alert how does it know which host triggered the alert and that it is a CPU alert? 

I am only marginally familiar with SNMP so if I am missing something obvious please enlighten me. Thanks